src/buttons/CopyToClipboard/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'preact/hooks'
+import PropTypes from 'prop-types'
+import CSS from './styles.css'
+
+import copyToClipboard from './../../utils/copy-to-clipboard'
+
+const CopyToClipboard = ({
+	dataTestId,
+	valueToCopy,
+	labelToCopy,
+	labelCopying,
+	labelCopied,
+	classToCopy,
+	classCopying,
+	classCopied,
+	copyingFeedbackTime,
+	copiedFeedbackTime,
+}) => {
+	const TO_COPY = { class: classToCopy, label: labelToCopy }
```

Comment:
(nit, naming) please avoid using the `ALL_CAPS` naming convention because it is not idiomatic JavaScript. That legacy convention (while originally intended to help improve clarity, in languages such as C, C++, etc.) actually reduces clarity as there is little to no consensus amongst JS developers on when/when-not to use it (thus, it has become a false-signal). Instead: please just use clear naming with standard JS `camelCase`, like so: 

```js
const toCopyClassAndLabel = { class: classToCopy, label: labelToCopy };
const copyingClassAndLabel = { class: classCopying, label: labelCopying }; // NOTE: Feel free to come up with a more descriptive/relevant name
const copiedClassAndLabel = { class: classCopied, label: labelCopied };
const copyTimeoutInMilliseconds = copyingFeedbackTime; // NOTE: please remove this line and the next one since they aren't necessary.
const copiedTimeoutInMilliseconds = copiedFeedbackTime;
```